### PR TITLE
Fix camera-relative previous camera position

### PR DESCRIPTION
**Pull request by EvgeniiG**
### Purpose of this PR
Fix camera-relative previous camera position

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2Ffix-prev-cam-pos&automation-tools_branch=master&unity_branch=trunk

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs [hunk 1]
```diff
@@ -568,15 +568,18 @@ void UpdateViewConstants(ref ViewConstants viewConstants, Matrix4x4 projMatrix,
...
                 // This fixes issue with cameraDisplacement stacking in prevViewProjMatrix when same camera renders multiple times each logical frame
                 // causing glitchy motion blur when editor paused.
                 if (m_LastFrameActive != Time.frameCount)
                 {
-                    Matrix4x4 cameraDisplacement = Matrix4x4.Translate(viewConstants.prevWorldSpaceCameraPos);
-                    viewConstants.prevViewProjMatrix *= cameraDisplacement; // Now prevViewProjMatrix correctly transforms this frame's camera-relative positionWS
+                    viewConstants.prevViewProjMatrix *= Matrix4x4.Translate(cameraDisplacement); // Now prevViewProjMatrix correctly transforms this frame's camera-relative positionWS
                 }
             }
+            }
```
Comment (fabien-unity) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs]: there's an extra curly brace showing up here

Comment (EvgeniiG) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs]: Bah, will fix. :-) Thanks guys

**Comment by EvgeniiG** [comment]
FYI Mac tests failed.

**Comment by sebastienlagarde** [comment]
Yes. Mac fail isnt related to this Pr

**Pull request merged**